cylc/flow/cfgspec/globalcfg.py

```diff
@@ -2084,6 +2085,11 @@ def _expand_platforms(self):
             self.sparse['platforms'] = expand_many_section(
```

Comment:
Do we not expand platform groups?

Reply:
We do not. 

Whilst I guess that it's concievably possible that you might want

```cylc
[platform groups]
  [[*]]
    [[[selection]]]
      method = definition order
  [[platform-one]]
    platforms = a, b
  [[platform-nine-and-three-quarters]]
    platforms = c, d
```

I'm not sure how useful this is for a setting with only two functional options.